nucypher/characters/chaotic.py

```diff
@@ -0,0 +1,263 @@
+import json
+from typing import Dict, Tuple
+
+from ferveo_py import DkgPublicParameters
+from nucypher_core import (
+    EncryptedThresholdDecryptionResponse,
+    SessionSecretFactory,
+    SessionStaticKey,
+    ThresholdDecryptionResponse,
+    ferveo,
+)
+
+from nucypher.characters.lawful import Bob, Enrico
+from nucypher.cli.types import ChecksumAddress
+from nucypher.crypto.powers import ThresholdRequestDecryptingPower
+from nucypher.network.decryption import ThresholdDecryptionClient
+from nucypher.policy.conditions.types import LingoList
+from nucypher.policy.conditions.utils import validate_condition_lingo
+
+
+class FakeNode:
+    def __init__(self, checksum_address):
+        self.checksum_address = checksum_address
+
+
+class _ParticipantKeyDict(dict):
+    def __init__(self, threshold_request_decrypting_power, *args, **kwargs):
+        self.threshold_request_decrypting_power = threshold_request_decrypting_power
+        super().__init__(*args, **kwargs)
+
+    def __getitem__(self, _item):
+        # Everybody has the same public key at the moment.
+        fifty_fiver = (
+            self.threshold_request_decrypting_power._get_static_secret_from_ritual_id(
+                55
+            )
+        )
+        return fifty_fiver.public_key()
+
+
+class _Blasphemy:
+    def __init__(
+        self, tau, threshold, shares_num, checksum_addresses, session_seed=None
+    ):
+        self.tau = tau
+        self.threshold = threshold
+        self.shares = shares_num
+        self.fake_nodes = [
+            FakeNode(checksum_address) for checksum_address in checksum_addresses
+        ]
+        self.checksum_addresses = checksum_addresses
+        if session_seed == None:
```

Comment:
This should be an identity comparison, not equality
```suggestion
        if session_seed is None:
```